### DISCUSSION TITLE
Set location for stored hashes

### DISCUSSION THREAD

**Discussion by madhatr** [Serene-Arc/bulk-downloader-for-reddit#944] [Ideas]
I download a subreddit and my script makes a new folder daily with the date. So all my hashes are always in that folder.  I would like to specify a folder so I can keep one mega hash file to compare against for every new day.

Currently I have to use a dupe finder later to delete any duped content.

I'm using this fork currently [bulk-downloader-for-reddit_with-saved-hashes)](https://github.com/thomas694/bulk-downloader-for-reddit_with-saved-hashes)

**Comment by Serene-Arc** [comment]
That repository is not this one. You'll have to go to that fork and ask them.

**Reply by madhatr** [reply]
Do you know how I do that when they don't have an issues or a discussion page? That I can see any how.

**Reply by Serene-Arc** [reply]
I don't know sorry but I can't help on a project that isn't mine.

**Comment by madhatr** [comment]
@thomas694 Not sure if this works, @'ing someone from another repo but are you able to assist here? Since it is your fork.

**Reply by thomas694** [reply]
Worked ;-)  I added your issue there.
